ddtrace/internal/forksafe.py

```diff
@@ -9,7 +9,7 @@
 log = logging.getLogger(__name__)
 
 
-_registry = []
+_registry = []  # type: typing.List[typing.Callable]
```

Comment:
Do we know the signature of the `Callable`? Can we add it to the type hint?

Reply:
From #2245, it seems that the signature of `_restart_on_hook` is empty, but I'm not sure if this would be universal for every hook added to `_registry`. Wdyt?

Reply:
Sounds about right, so `Callable[[], None]`

Reply:
Bump on this one - looks like we still need to update this hint